csfieldguide/templates/interactives/regular-expression-filter.html

```diff
@@ -0,0 +1,56 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+  <div class="container">
+    <div class="row">
+      <div class="col-12">
+        <div id="interactive-regular-expression-filter">
+          <h3>{% trans "Word Filter" %}</h3>
+
+          <button id="interactive-regular-expression-filter-load" class="btn btn-primary">{% trans "Load word file (2.8MB)" %}</button>
+
+          <div id="interactive-regular-expression-filter-loading-progress">
+            <h6>{% trans "Loading word file - may take longer than 10 seconds." %}</h6>
+          </div>
+
+          <div class="row justify-content-center">
+            <div class="col-6"  id="interactive-regular-expression-filter-controls">
+              <h5>{% trans "Regular Expression" %}</h5>
+              <input id="interactive-regular-expression-filter-regex" type="text" placeholder="Enter expression here" class="form-control mb-2">
+              <button id="interactive-regular-expression-filter-filter" class="btn btn-primary btn-expand">{% trans "Filter words" %}</button>
+
+              <p id="interactive-regular-expression-filter-feedback"></p>
+            </div>
+
+            <div class="col-3">
+              <p id="interactive-regular-expression-filter-words" class="text-left">
+              </p>
+            </div>
+          </div>
+
+          <a id="interactive-regular-expression-filter-link" href="./"></a>
+        </div>
+      </div>
+    </div>
+  </div>
+{% endblock html %}
+
+{% block css %}
+  <link rel="stylesheet" href="{% static 'interactives/regular-expression-filter/css/regular-expression-filter.css' %}">
+{% endblock css %}
+
+{% block js %}
+  <script type="text/javascript">
+    var words_file_location = "{% static 'interactives/regular-expression-filter/en-words.txt' %}";
+    var reg_exp_filter_il8n = {
+      "showing": "{% trans 'Showing ' %}",
+      "out-of": "{% trans ' out of ' %}",
+      "words": "{% trans ' lower case words' %}",
```

Comment:
These three strings should be translated as one, to allow different ordering of words in other languages.